lib/money/money.rb

```diff
@@ -390,11 +415,27 @@ def symbol
     currency.symbol || "¤"
   end
 
+  def infinite_precision?
+    @infinite_precision
+  end
+
   # Common inspect function
   #
   # @return [String]
   def inspect
-    "#<#{self.class.name} fractional:#{fractional} currency:#{currency}>"
+    "#<#{self.class.name} fractional:#{fractional} currency:#{currency}"
```

Comment:
missing `>` at the end. Also should not show up in the diff at all

Reply:
I can clean that up.